src/Symfony/Component/Mime/RawMessage.php

```diff
@@ -28,6 +28,26 @@ public function __construct($message)
         $this->message = $message;
     }
 
+    /**
+     * @param string|iterable $wrappedMessage
+     *
+     * @return $this
+     */
+    public function setMessage($wrappedMessage)
+    {
+        $this->message = $wrappedMessage;
+
+        return $this;
+    }
+
+    /**
+     * @return iterable|string
+     */
+    public function getMessage()
+    {
+        return $this->message;
+    }
```

Comment:
These two methods do not seem to be used anywhere in this PR, can we remove them?